app/main/views/g_cloud.py

```diff
@@ -513,16 +513,33 @@ def view_project(framework_family, project_id):
         "data_value": current_project_stage
     }]
 
-    return render_template('direct-award/view-project.html',
-                           framework=framework,
-                           project=project,
-                           custom_dimensions=custom_dimensions,
-                           search=search,
-                           buyer_search_page_url=buyer_search_page_url,
-                           search_summary_sentence=search_summary_sentence,
-                           framework_urls=framework_urls,
-                           call_off_contract_url=framework_urls['call_off_contract_url'],
-                           project_outcome_label=project_outcome_label)
+    try:
+        following_framework = framework_helpers.get_framework_or_500(
+            data_api_client,
+            content_loader.get_metadata(framework['slug'], 'following_framework', 'slug'),
+            current_app.logger,
+        )
+    except ContentNotFoundError:
+        following_framework = None
+
+    temporary_message_status = get_saved_search_temporary_message_status(
+        project, framework, following_framework
+    ) if following_framework else None
+
+    return render_template(
+        'direct-award/view-project.html',
+        framework=framework,
+        following_framework=following_framework,
+        project=project,
+        custom_dimensions=custom_dimensions,
+        search=search,
+        buyer_search_page_url=buyer_search_page_url,
+        search_summary_sentence=search_summary_sentence,
+        framework_urls=framework_urls,
+        call_off_contract_url=framework_urls['call_off_contract_url'],
```

Comment:
Weird that we pass this through separately when we pass all the framework urls through in the line above.

Not your code though.